src/odemis/driver/actuator.py

```diff
@@ -666,6 +667,27 @@ def _get_rot_matrix(self, invert=False):
             rotation *= -1
         return RigidTransform(rotation=rotation).matrix
 
+    def _getAxesRange(self):
```

Comment:
Actually, it's not a really good name, because by convention functions that are called `get*` are supposed to return a value... and not change anything else. I'd suggest to call it `_updateAxesRange`.

Reply:
I have renamed the function.